src/Sulu/Bundle/PageBundle/Domain/Event/PageCreatedEvent.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\PageBundle\Domain\Event;
+
+use Sulu\Bundle\EventLogBundle\Domain\Event\DomainEvent;
+use Sulu\Bundle\PageBundle\Admin\PageAdmin;
+use Sulu\Bundle\PageBundle\Document\BasePageDocument;
+use Sulu\Component\Content\Document\Behavior\SecurityBehavior;
+
+class PageCreatedEvent extends DomainEvent
+{
+    /**
+     * @var BasePageDocument
+     */
+    private $pageDocument;
+
+    /**
+     * @var string
+     */
+    private $locale;
+
+    /**
+     * @var mixed[]
+     */
+    private $payload;
+
+    /**
+     * @param mixed[] $payload
+     */
+    public function __construct(
+        BasePageDocument $pageDocument,
+        string $locale,
```

Comment:
I would prefer to be specific here, because in 3.0, there won't be a `BasePageDocument` with a `getLocale()` method anymore, right? And it would also be consistent with the other domain events for localized entities